dockerfiles/execute/run_migrations.sh

```diff
@@ -29,6 +29,8 @@ fi
 # Run the DB migrations
 echo "Connecting with: $VDB_PG_CONNECT"
 ./goose -dir db/migrations postgres "$VDB_PG_CONNECT" up
+psql $VDB_PG_CONNECT -c 'CREATE SCHEMA IF NOT EXISTS maker;'
+./goose -table maker.goose_db_version -dir db/vdb-mcd-transformers/migrations postgres "$VDB_PG_CONNECT" up
```

Comment:
So just to make sure I understand - we need some vdb to image to run its migrations and setup the public schema before this fires, right? Since we're not loading the public schema? (Totally makes sense if so, since I think we prob don't want to be loading the public schema here, just thinking about order dependency in docker-compose etc)

Reply:
Line 31 above loads the public schema, just like before. If we want to take it out I'd be okay with that - when we run the docker compose setup there are two containers trying to migrate public. One "wins" and it works itself out with retries, but I don't like it.

Reply:
oh snap I didn't realize there were two `up` statements here 👍